chess_board.js

```diff
@@ -0,0 +1,54 @@
+
+    var div = document.createElement('div');
+    div.className = "container";
+    document.body.appendChild(div);
+
+    for (i = 0; i < 3; i++) {
+        var block = document.createElement('div');
+        block.className = "block";
+        div.appendChild(block);
+    }
+
+    var blockList = document.getElementsByClassName('block');
+
+    for (i = 0; i < 3; i++) {
+        if (i === 0 || i === 2) {
+            for (j = 0; j < 8; j++) {
+                var row = document.createElement('div');
+                row.className = "row";
+                row.innerHTML = (8 - j);
+                blockList[i].appendChild(row);
+            }
+        }
+    }
+
+    for (i = 0; i < 10; i++) {
+        if (i === 0 || i === 9) {
+            var chessrow = document.createElement('div');
+            chessrow.className = "chessrow";
```

Comment:
1. Почему-то везде разные кавычки
2. Рекомендую свойство classList
```suggestion
            chessrow.className = 'chessrow';
```